main.cpp

```diff
@@ -19,27 +19,27 @@ using namespace std;
 
 struct Person
 {
-int age;
-int height;
-float hairLength;
-float GPA;
-unsigned int SATScore;
-int distanceTraveled;
-
-    struct Foot
-    {
-        int numSteps = 0;
-        void stepForward();
-        int stepSize();
-    };
-
-void run(int howFast, bool startWithLeftFoot, Foot leftFoot, Foot rightFoot);
+    int age;
+    int height;
+    float hairLength;
+    float GPA;
+    unsigned int SATScore;
+    int distanceTraveled;
+
+        struct Foot
+        {
+            int numSteps = 0;
+            void stepForward();
+            int stepSize();
+        };
+
+    void run(int howFast, bool startWithLeftFoot, Foot leftFoot, Foot rightFoot);
 
 };
```

Comment:
the `Foot` class should only be indented once.  it is indented twice.  It should be aligned with the other member variables. 
- [ ] revise